korangar/src/graphics/renderers/picker/mod.rs

```diff
@@ -83,6 +90,20 @@ impl PickerRenderer {
     ) {
         self.tile_renderer.render(render_target, render_pass, camera, vertex_buffer);
     }
+
+    pub fn dispatch_selector(
+        &self,
+        render_target: &mut <PickerRenderer as Renderer>::Target,
+        compute_pass: &mut ComputePass,
+        window_size: ScreenSize,
+        pointer_position: ScreenPosition,
+    ) {
+        let clamped_pointer_position = Vector2::new(
+            pointer_position.left.clamp(0.0, window_size.width) as u32,
+            pointer_position.top.clamp(0.0, window_size.height) as u32,
+        );
```

Comment:
Just curious, do we really want to clamp the selection here or should we rather not dispatch anything if the cursor position is not in bounds?